stories/CatalogHelper.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import CatalogHelper from '../src/helpers/CatalogHelper';
+import {Kpi} from '../src/components/simple/Kpi';
```

Comment:
Formatting here is a little of. Could you add spaces around `kpi` and `storiesOf`?

Reply:
Fixed. 
btw, how come that linter doesnt warn here?